dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -1125,6 +1128,9 @@ Sedp::Task::svc_i(const ParticipantData_t* ppdata)
   if (spdp_->available_builtin_endpoints() & DISC_BUILTIN_ENDPOINT_PARTICIPANT_ANNOUNCER &&
       avail & BUILTIN_ENDPOINT_PARTICIPANT_MESSAGE_DATA_READER) {
     DCPS::AssociationData peer = proto;
+    if (beq & BEST_EFFORT_PARTICIPANT_MESSAGE_DATA_READER) {
```

Comment:
should similar logic apply to the secure version of the participant message data reader?

Reply:
Yes, see DDS Security 1.1 section 7.4.2:
```
The QoS associated with the DCPSParticipantMessageSecure Topic shall be the same as for the
DCPSParticipantMessage Topic as defined in sub clause 8.4.13 of the DDS Interoperability Wire
Protocol [2]
```